docs/guides/developer/docs/api/usage.md

```diff
@@ -10,16 +10,20 @@ Since the API is versioned, it supports specification of a version identifier as
 
 Header            |Type       | Comment                                                                   
 :-----------------|:----------|:--------------------------------------------------------------------------
-`Accept`          |`String`   | The format is specified to use `application/<version>`, or `application/<version>+<format>` to also specify the required format.
+`Accept`          |`String`   | `application/<version>+<format>` is used to specify the API version and format
+
+Notes:
+
+- The API currently only supports the format [JSON][3]
 
 __Sample__
 ```
 Accept: application/v1.0.0+json
 ```
 
-If that header is not specified, or no version information can be extracted from the header, the assumption is that the request should be executed against the most recent version. If the version specified is not available, `400 (BAD REQUEST)` is returned as the HTTP response code.
+If that header is not specified, or no version information can be extracted from the header, the assumption is that the request should be executed against the most recent version. If the version specified is not available, `406 (NOT ACCEPTABLE)` is returned as the HTTP response code.
```

Comment:
stauber$ curl -I -H "Accept: application/v1.2.0+json" -user:pwd <admin>/api/info/me
HTTP/1.1 406 Not Acceptable
[...]